wrangler-core/src/main/java/co/cask/directives/xml/XmlParser.java

```diff
@@ -102,14 +103,20 @@ public void destroy() {
     return rows;
   }
 
+  @Override
+  public MutationDefinition lineage() {
+    MutationDefinition.Builder builder = new MutationDefinition.Builder(NAME);
+    builder.addMutation(column, MutationType.MODIFY);
```

Comment:
I believe XmlParser adds to the same column, so since no new columns are being made it's modify.